media-plugins/vdr-dvbapi/vdr-dvbapi-2.2.4.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit vdr-plugin-2
+
+DESCRIPTION="VDR Plugin: allows connect VDR to OScam"
+HOMEPAGE="https://github.com/manio/vdr-plugin-dvbapi"
+SRC_URI="https://github.com/manio/vdr-plugin-dvbapi/archive/v${PV}.tar.gz -> vdr-dvbapi-${PV}.tar.gz"
```

Comment:
Having different filename and directory inside it is confusing. Since you have full control over the former, just make it `vdr-plugin-dvbapi-${PV}` to match. You may want to declare it as `MY_P` above and reuse the var in both places.